fe1-web/src/features/events/components/EventList.tsx

```diff
@@ -1,159 +1,137 @@
-
-    return {
-      pastEvents: newPastEvents,
-      currentEvents: newCurrentEvents,
-      futureEvents: newFutureEvents,
-    };
-  });
+    upcomingEvents: EventState[];
+  }>(() => categorizeEventsByTime(Timestamp.EpochNow(), events));
 
-  const [showUpcoming, setShowUpcoming] = useState(true);
   const [showCurrent, setShowCurrent] = useState(true);
-  const [showPast, setShowPast] = useState(false);
+  const [showPast, setShowPast] = useState(currentEvents.length === 0);
 
   useEffect(() => {
-    const interval = setInterval(() => {
-      const [newPastEvents, newCurrentEvents, newFutureEvents] = categorizeEventsByTime(
-        Timestamp.EpochNow(),
-        events,
-      );
-
-      setEvents({
-        pastEvents: newPastEvents,
-        currentEvents: newCurrentEvents,
-        futureEvents: newFutureEvents,
-      });
-    }, 1000);
+    const interval = setInterval(
+      () => setEvents(categorizeEventsByTime(Timestamp.EpochNow(), events)),
+      1000,
+    );
 
     // clear the interval when unmouting the component
     return () => clearInterval(interval);
   }, [events]);
 
   return (
-    <View style={List.container}>
-      <ListItem.Accordion
-        containerStyle={List.accordionItem}
-        style={List.accordionItem}
-        content={
-          <ListItem.Content>
-            <ListItem.Title style={[Typography.base, Typography.important]}>
-              {STRINGS.events_list_upcoming} ({futureEvents.length})
-            </ListItem.Title>
-          </ListItem.Content>
-        }
-        isExpanded={showUpcoming}
-        onPress={() => setShowUpcoming(!showUpcoming)}>
-        {futureEvents.map((event, idx) => (
-          <EventListItem
-            key={event.id}
-            eventId={event.id}
-            eventType={event.eventType}
-            isFirstItem={idx === 0}
-            isLastItem={idx === futureEvents.length - 1}
-            testID={`upcoming_event_selector_${idx}`}
-          />
-        ))}
-      </ListItem.Accordion>
-      <ListItem.Accordion
-        containerStyle={List.accordionItem}
-        style={List.accordionItem}
-        content={
-          <ListItem.Content>
-            <ListItem.Title style={[Typography.base, Typography.important]}>
-              {STRINGS.events_list_current} ({currentEvents.length})
-            </ListItem.Title>
-          </ListItem.Content>
-        }
-        isExpanded={showCurrent}
-        onPress={() => setShowCurrent(!showCurrent)}>
-        {currentEvents.map((event, idx) => (
-          <EventListItem
-            key={event.id}
-            eventId={event.id}
-            eventType={event.eventType}
-            isFirstItem={idx === 0}
-            isLastItem={idx === currentEvents.length - 1}
-            testID={`current_event_selector_${idx}`}
-          />
-        ))}
-      </ListItem.Accordion>
-      <ListItem.Accordion
-        containerStyle={List.accordionItem}
-        style={List.accordionItem}
-        content={
+    <View>
```

Comment:
later on we might want to address the cases when: 
1. all the events are "upcoming"
or
2. there are no events at all